website/www/site/data/capability_matrix.yaml

```diff
@@ -393,7 +393,7 @@ capability-matrix:
             - class: dataflow
               l1: "Partially"
               l2: non-merging windows
-              l3: State is supported for non-merging windows. The MapState and SetState classes are supported with streaming pipelines that use Streaming Engine and version 2.58.0 or later of the Java SDK. SetState and MapState are not supported for pipelines that use Runner v2.
+              l3: State is supported for non-merging windows. The MapState and SetState classes and the MultimapState interface are supported in the following scenarios: Java pipelines that don't use Streaming Engine; Java pipelines that use Streaming Engine and version 2.58.0 or later of the Java SDK. SetState, MapState, and MultimapState are not supported for pipelines that use Runner v2.
```

Comment:
we plan to add support for runner v2, should we say "not yet supported"?

Reply:
We're not supposed to include any information about future features in the technical documentation, because if anything changes, we could open ourselves to legal liability, or just generally frustrate customers. Here's the guidance for the gcp docs, which also applies here: https://developers.google.com/style/future